Modules/@babylonjs/react-native/package.json

```diff
@@ -27,7 +27,7 @@
     "base-64": "^0.1.0"
   },
   "peerDependencies": {
-    "@babylonjs/core": "^4.2.0",
+    "@babylonjs/core": "^5.0.0-alpha.6",
```

Comment:
I guess it doesn't really matter that much, but didn't you say the min ver is 5.0.0-alpha.1?

Reply:
I forgot there's actually a change needed in alpha.6 for tearing down the XR session updating the description.